tests/e2e/scale/upgrade/test_upgrade_with_scaled_pvcs_pods.py

```diff
@@ -0,0 +1,142 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import constants, scale_lib
+from ocs_ci.utility import utils, templating
+from ocs_ci.ocs.scale_lib import FioPodScale
+from ocs_ci.utility.utils import ocsci_log_path
+from ocs_ci.ocs.resources.pvc import get_all_pvcs
+from ocs_ci.framework.pytest_customization.marks import (
+    pre_ocs_upgrade,
+    post_ocs_upgrade,
+    skipif_bm,
+    skipif_external_mode,
+)
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+
+log = logging.getLogger(__name__)
+
+# Scaling PVC count
+SCALE_COUNT = 1500
+# Each pod will be created with 20 PVCs attached
+PVCS_PER_POD = 20
+# Scale data file
+log_path = ocsci_log_path()
+SCALE_DATA_FILE = f"{log_path}/scale_data_file.yaml"
+
+
+@skipif_external_mode
+@skipif_bm
+@pre_ocs_upgrade
```

Comment:
im not sure this marker is correct - do we want to run this test for each upgrade job? in my opnion, new marker is needed here
@petr-balogh What do you think?

Reply:
Changed marker to pre_upgrade